.solr_wrapper

```diff
@@ -0,0 +1,12 @@
+# Place any default configuration for solr_wrapper here
+# port: 8983
+
+download_dir: tmp/solr_wrapper
```

Comment:
I'm :-1: on including `download_dir` and `instance_dir`. I'd prefer people handle this locally with their own config files.